tdp/cli/utils.py

```diff
@@ -129,5 +148,6 @@ def vars(func: FC) -> FC:
         envvar="TDP_VARS",
         required=True,
         type=click.Path(resolve_path=True, path_type=Path),
+        callback=_check_vars_existence,
```

Comment:
Using `exists=True` does not work instead of your custom callback ?
```suggestion
        type=click.Path(exists=True, resolve_path=True, path_type=Path),
```

Reply:
You're right, it would be even simpler!